src/backend/joanie/tests/core/test_models_product.py

```diff
@@ -126,4 +126,4 @@ def test_models_product_target_course_runs_property(self):
         with self.assertNumQueries(1):
             course_runs = product.target_course_runs.order_by("pk")
             self.assertEqual(len(course_runs), 3)
-            self.assertListEqual(list(course_runs), [cr1, cr2, cr3])
+            self.assertCountEqual(list(course_runs), [cr1, cr2, cr3])
```

Comment:
Ditto

Reply:
I've used assertCountEqual on purpose not to have to worry about the order as getting ride of the id with an auto-increment has changed the order